### PR TITLE
Composite pricing, updated (supersedes #830)

### PR DESCRIPTION
**Pull request by mfhepp**
This pull request updates #830. It adds examples for two domains and addresses the issue described in #829.

CompoundPriceSpecification is a needed enhancement from GoodRelations for cases in which a price consists of multiple components (e.g. 50 USD per day and 0.10 USD per mile for a rental car).

### CODE REVIEW

File: data/sdo-examples-goodrelations.txt [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+PRE-MARKUP:
+
+A bank account fee will be 5 USD per month and $ 0.10 per transaction
+
+MICRODATA:
+
+<div itemscope itemtype="http://schema.org/Offer">
+  <h1 itemprop="description">Our Private Bank Account Package: Low monthly fees and fair per-use charges.</h1>
+    <b itemprop="name">Private Bank Account Package</b>
+    <span itemprop="itemOffered" itemscope itemtype="BankAccount">
```
Comment (unor) [inline data/sdo-examples-goodrelations.txt]: The `itemtype` value must be an absolute URL.

File: data/sdo-examples-goodrelations.txt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        <span itemprop="name">ACME Private Bank Account</span>
+        APR: <meta itemprop="annualPercentageRate" content="0.0314">3.14%
+    </span> 
+  </div>
+  <span itemprop="priceSpecification" itemscope itemtyp="http://schema.org/CompoundPriceSpecification">
```
Comment (unor) [inline data/sdo-examples-goodrelations.txt]: Typo: `itemtyp` → `itemtype`

File: data/sdo-examples-goodrelations.txt [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      <span itemprop="priceComponent" itemscope itemtype="http://schema.org/UnitPriceSpecification">
+          <span itemprop="price">5</span> 
+          <meta itemprop="priceCurrency" content="USD"> US$
+          <meta itemprop="unitCode" content="MON"> per month
+      </span>
+      <span itemprop="priceComponent" itemscope itemtype="http://schema.org/UnitPriceSpecification">
+          <span itemprop="price">0.10</span> 
+          <meta itemprop="priceCurrency" content="USD"> US$
+          <span itemprop="unitText"> per transaction
```
Comment (unor) [inline data/sdo-examples-goodrelations.txt]: might miss a closing `</span>`

File: data/sdo-examples-goodrelations.txt [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+PRE-MARKUP:
+
+A rental car is 50 USD per day and and $ 0.10 per mile.
+
+MICRODATA:
+
+<div itemscope itemtype="http://schema.org/Offer">
+  <h1 itemprop="description">Our standard rental car is 50 USD per day and and $ 0.10 per mile.</h1>
+    <b itemprop="name">Rental Car Package</b>
+    <span itemprop="itemOffered" itemscope itemtype="Car">
```
Comment (unor) [inline data/sdo-examples-goodrelations.txt]: The `itemtype` value must be an absolute URL.

File: data/sdo-examples-goodrelations.txt [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+        <span itemprop="name">Volkswagen Golf Class Car</span>
+    </span> 
+  </div>
+  <span itemprop="priceSpecification" itemscope itemtyp="http://schema.org/CompoundPriceSpecification">
```
Comment (unor) [inline data/sdo-examples-goodrelations.txt]: Typo: `itemtyp` → `itemtype`

**Comment by danbri** [comment]
Merged as-is, but @mfhepp if you could fold in @unor 's various fixes into a followup PR that would be great.

**Comment by unor** [comment]
I submitted a PR for the mentioned fixes as well as two fixes for issues I didn’t notice when reviewing it the first time: https://github.com/schemaorg/schemaorg/pull/1038

**Comment by mfhepp** [comment]
Thanks!

**Pull request merged**